homeassistant/components/plex/media_browser.py

```diff
@@ -52,14 +52,71 @@ class UnknownMediaType(BrowseError):
 _LOGGER = logging.getLogger(__name__)
 
 
-def browse_media(
-    entity_id, plex_server, media_content_type=None, media_content_id=None
-):
+def browse_media(entity, is_internal, media_content_type=None, media_content_id=None):
     """Implement the websocket media browsing helper."""
 
+    def item_payload(item):
+        """Create response payload for a single media item."""
+        try:
+            media_class = ITEM_TYPE_MEDIA_CLASS[item.type]
+        except KeyError as err:
+            _LOGGER.debug("Unknown type received: %s", item.type)
+            raise UnknownMediaType from err
+        payload = {
+            "title": item.title,
+            "media_class": media_class,
+            "media_content_id": str(item.ratingKey),
+            "media_content_type": item.type,
+            "can_play": True,
+            "can_expand": item.type in EXPANDABLES,
+        }
+        if hasattr(item, "thumbUrl"):
+            entity.plex_server.thumbnail_cache.setdefault(
+                str(item.ratingKey), item.thumbUrl
+            )
+
+            if is_internal:
+                thumbnail = item.thumbUrl
+            else:
+                thumbnail = entity.get_browse_image_url(
+                    item.type, item.ratingKey, item.ratingKey
```

Comment:
```suggestion
                    item.type, item.ratingKey
```
The third parameter is optional, so rather than passing `item.ratingKey` twice, just don't use it.

Reply:
I could have sworn I was hitting some testing issues without this param, but seems fine now. 🤷